ruby/sinatra/app.rb

```diff
@@ -0,0 +1,77 @@
+
+require 'bundler/setup'
+
+require 'connection_pool'
+require 'date'
+require 'json'
+require 'redis'
+require 'sinatra'
+require 'sinatra/json'
+
+require 'ddtrace'
+require 'ddtrace/contrib/sinatra/tracer'
+
+require './post.rb'
+
+class App < Sinatra::Application
+  REDIS_HOST = '127.0.0.1'.freeze()
+  REDIS_PORT = 6379
+
+  configure do
+    settings.datadog_tracer.configure(default_service: 'sinatra-demo',
```

Comment:
ENV also for names; we may need to deploy this application many times to check compatibility of different versions